content/en/project/_index.md

```diff
@@ -10,17 +10,21 @@ aliases:
 
-Because Interlisp has been around a long time, with multiple iterations, you might find the multitude of names for the Interlisp language and related companies confusing. To assist, we have created a [glossary of terms](glossary) used on this website.
+Because of the long history of naming and renaming of groups, software, environments, machines, languages, we have created a [glossary of terms](glossary).
 
-## What are the origins of Interlisp? Why do we care?
+## What are the origins of Medley and Interlisp?
 
-The 1970s and 80s saw major advances in computing and Human-Computer interaction.  Interlisp was one of the many experimental systems developed during these heady times.  It represents an early example of a system with a GUI (Graphical User Interface), and IDE (Integrated Development Environment).
+The 1970s and 80s saw major advances in computing and Human-Computer interaction.  At Xerox PARC there were four IDEs developed with different approaches: Smalltalk, with an object-oriented system; Interlisp, for researchers in AI and exploratory development; Cedar/Mesa, with a strongly typed language for software engineering. They each had a different model for source code management, development and versioning. In common they shared an infrastructure and vision computing and distributed systems, with support for Ethernet networking, printing, file servers and network protocols. Each environment had its own Virtual Machine instruction set tuned for the language "all the way down", in a single address space, in a way that opened the systems to customization.
 
-Within the Xerox PARC environment, it was an early example of a network enabled machine supporting email, file sharing, printing and more.  In essence it was a forerunner of today's computing systems.
+There were several different [D-machines](http://www.bitsavers.org/pdf/xerox/parc/Exploring_the_Ethernet_with_Mouse_and_Keyboard_May81.pdf): Dorado, Dolphin, Dandelion, Daybreak, and others. Each environment had its own microcode to implement its instruction set; once loaded, each would take over the machine -- there was no OS. Each had large-screen black and white CRTs for a graphical user interface, mouse and cursor pointer, high quality fonts.
 
-Medley Interlisp represents the last Xerox Interlisp release.  Medley continued Xerox PARC's experimentation with developing systems for [D-machines](http://www.bitsavers.org/pdf/xerox/parc/Exploring_the_Ethernet_with_Mouse_and_Keyboard_May81.pdf). A family of computers (Dorado, Dolphin, Dandelion, Daybreak) resulted. The project was later spun out to a company called Envos, which then turned into Venue.
+Smalltalk has evolved considerably since PARC through ParcPlace Systems and other ways. [The SmallTalk Zoo]. Cedar/Mesa has an emulator of its instruction set for Sun SPARC. 
```

Comment:
It may not be worth mentioning in this text, but PCedar's compilation process generated C code (for big-endian machines) which was loaded into the running system.  It knew about the Solaris SPARC ABI, and was never ported to compile for little-endian systems.